webapp/tests/mocha/unit/swRegister.spec.js

```diff
@@ -47,6 +46,8 @@ describe('Service worker registration (swRegister.js)', () => {
       expect(actual).to.be.an('object');
       expect(callback.callCount).to.eq(1);
       done();
+    }).catch(() => {
+      process.exit(1);
```

Comment:
Can we return the `swRegister` call and remove the done as you did above? That'll fail the test gracefully.

Reply:
`executeSwLifecycle` is called after the `swRegister` so will throw the error in the catch block.